server/service/src/requisition/common.rs

```diff
@@ -102,3 +105,41 @@ impl From<RepositoryError> for OrderTypeNotFoundError {
         Self::DatabaseError(error)
     }
 }
+
+pub struct IndicatorGenerationInput<'a> {
```

Comment:
Why did you put lifelines on? 🤔

Reply:
I had thought you needed a lifetime to ensure referenced components outlive the struct.

Would this be preferable with cloning?

```
pub struct IndicatorGenerationInput {
    pub store_id: String,
    pub period_id: String,
    pub program_indicators: Vec<ProgramIndicator>,
    pub other_party_id: String,
}
```

Reply:
I think cloning is fine for something this small (: